text/0000-roadmap-2021.md

```diff
@@ -0,0 +1,129 @@
+- Feature Name: n/a
+- Start Date: 2020-12-03
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes three parallel efforts for completion in 2021:
```

Comment:
It would be helpful to have more prose explaining the overall theme for this year's roadmap which seems to be exclusively about project health.